src/NosCore.Parser/Parsers/NpcMonsterParser.cs

```diff
@@ -0,0 +1,495 @@
+﻿//  __  _  __    __   ___ __  ___ ___
+// |  \| |/__\ /' _/ / _//__\| _ \ __|
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// 
+// Copyright (C) 2019 - NosCore
+// 
+// NosCore is a free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using NosCore.Core;
+using NosCore.Core.I18N;
+using NosCore.Data.Enumerations.I18N;
+using NosCore.Data.Enumerations.Map;
+using NosCore.Data.StaticEntities;
+using NosCore.Parser.Parsers.Generic;
+using Serilog;
+
+namespace NosCore.Parser.Parsers
+{
+    //    VNUM	{NpcMonsterVNum}
+    //    NAME {NameI18NKey}
+
+    //    LEVEL	{Level}
+    //    RACE	{Race}	{RaceType}
+    //    ATTRIB	{Element}	{ElementRate}	{FireResistance}	{WaterResistance}	{LightResistance}	{DarkResistance}
+    //    HP/MP	{HP}	{MP}
+    //    EXP	{XP}	{JXP}
+    //    PREATT	{IsHostile}	{NoticeRange}	{Speed}	{RespawnTime}	400
+    //    SETTING	0	0	-1	0	0	0
+    //    ETC	8	1	0	0	0	0	0	0
+    //    PETINFO	1	10	0	50
+    //    EFF	200	0	0
+    //    ZSKILL	0	1	3	2	12	0	0
+    //    WINFO	0	0	0
+    //    WEAPON	16	1	0	0	0	11	-20
+    //    AINFO	0	0
+    //    ARMOR	{CloseDefence}	{DistanceDefence}	{MagicDefence}	{DefenceDodge}	{DistanceDefenceDodge}
+    //    SKILL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+    //    PARTNER	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+    //    BASIC	0	0	4	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+    //    CARD	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+    //    MODE	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+    //    ITEM	2000	9000	1	16	800	1	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0	-1	0	0
+    //#========================================================
+
+    public class NpcMonsterParser
+    {
+        private const string FileNpcId = "\\monster.dat";
+        private readonly IGenericDao<BCardDto> _bCardDao;
+        private readonly IGenericDao<DropDto> _dropDao;
+        private readonly ILogger _logger;
+        private readonly IGenericDao<NpcMonsterDto> _npcMonsterDao;
+        private readonly IGenericDao<NpcMonsterSkillDto> _npcMonsterSkillDao;
+        private readonly IGenericDao<SkillDto> _skillDao;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_skillDao' or refactor the code to use its value.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4512441)